docs/releasenotes/index.md

```diff
@@ -9,6 +9,18 @@ has_children: true
 
 [Full release notes](https://github.com/hyperledger/firefly/releases)
 
+## [v1.3.0 - February XX, 2024](https://github.com/hyperledger/firefly/releases/tag/v1.1.0)
+
+[Migration guide](1.3_migration_guide.md)
+
+What's New:
+
+- Namespaces now have individual event streams
```

Comment:
Not sure I would list this one first. It's not a very exciting feature to the end-user.